pages/glossary/keyboard-actionable-element.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Keyboard Actionable Element
+key: keyboard-actionable-element
+unambiguous: true
+objective: true
+input_aspects:
+  - Accessibility tree
+  - CSS styling
+  - DOM tree
+---
+
+An HTML element is _keyboard actionable_ if all the following are true:
+
+- the element is part of [sequential focus navigation][]; and
+- the element is [visible][] when [focused][]; and
+- the element can be [activated][]; and
```

Comment:
Do we need the "the element can be [activated][]" condition? If it can be activated by keyboard, then it can be activated.